src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -145,17 +151,31 @@ public String getInstanceId() {
   }
 
   public String getCallNumber() {
-    return getProperty(holdingRepresentation, "callNumber");
+    return getEffectiveCallNumbers(CALL_NUMBER_KEY);
   }
 
   public String getCallNumberPrefix() {
-    return getProperty(holdingRepresentation, "callNumberPrefix");
+    return getEffectiveCallNumbers(CALL_NUMBER_PREFIX_KEY);
   }
 
   public String getCallNumberSuffix() {
-    return getProperty(holdingRepresentation, "callNumberSuffix");
+    return getEffectiveCallNumbers(CALL_NUMBER_SUFFIX_KEY);
+  }
+
+  private String getEffectiveCallNumbers(String propertyName) {
```

Comment:
This method seems to be used for call numbers, suffixes and prefixes. The name suggests it is only used for call numbers, could this be renamed to clarify it's broader usage

Reply:
Fix